nservicebus/upgrades/5to6/container.md

```diff
@@ -54,4 +54,14 @@ snippet: 5to6-ExplicitProperties
 
 To access it at runtime create a new [`Feature`](/nservicebus/pipeline/features.md) and put the following code in the `.Setup` method
 
-snippet: 5to6-IConfigureComponentsNotInjected
\ No newline at end of file
+snippet: 5to6-IConfigureComponentsNotInjected
+
+## Existing passed to the configuration API are no longer disposed
+
+snippet:Autofac_Existing
```

Comment:
@danielmarbach so thi is problematic since it will also show the v3 and v4 in the upgrade guide. the convention is to create a new snippet that just has the versions u want